src/doc_table.c

```diff
@@ -345,6 +348,7 @@ void DocTable_Free(DocTable *t) {
       DMD_Return(md);
     }
   }
+  t->memsize -= t->cap * sizeof(DMDChain);
```

Comment:
Why update this?

Reply:
We are freeing `t->buckets` on next line

Reply:
We are freeing `t` in general.. No one will use this table from here on no?

Reply:
🤦 You are right. Thanks.